tests/zero_code_change/test_tensorflow2_integration.py

```diff
@@ -52,7 +52,7 @@ def helper_test_keras_v2(script_mode: bool = False, eager_mode: bool = True):
     tf.keras.backend.clear_session()
     if not eager_mode:
         tf.compat.v1.disable_eager_execution()
-    with SagemakerSimulator() as sim:
+    with SagemakerSimulator(disable_tb=True) as sim:
```

Comment:
do we need this in gradienttape tests? and other tf2 tests that use sagemaker simulator?

Reply:
This is a temporary workaround since the DLC team is currently facing issues with long running jobs.

The timeout issue currently occurs only with :
`tests/zero_code_change/test_tensorflow2_integration.py `

By disabling tb_writer in this test, we reduce the running time of this test.